pkg/cmd/roachprod/main.go

```diff
@@ -778,12 +778,14 @@ const (
 	customizedApp = `
   customized - Cockroach customized builds, usually generated by running
                ./scripts/tag-custom-build.sh. Must provide a specific tag.`
+	localApp = `
+  local      - Use a provided local binary, must provide the path to the binary.`
 	workloadApp = `
   workload   - Cockroach workload application.`
 )
 
 var deployCmd = &cobra.Command{
-	Use:   "deploy <cluster> <application> <version>",
+	Use:   "deploy <cluster> <application> <version>/<binary-path>",
```

Comment:
Nit: use `|` to denote an alternative; also `binary-path` -> `pathToBinary`

Reply:
Done.